src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -1610,9 +1592,27 @@ await _jobMonitor.Begin(new UnmanageGameJob(installation), async ctx =>
         }
 
         var result = await tx.Commit();
+        var newLoadoutId = result[entityIdList[loadout.Id]];
+
+        // Generate a new name and short name
+        var newShortName = LoadoutNameProvider.GetNewShortName(Loadout.All(baseDb)
+            .Where(l => l.IsVisible() && l.InstallationId == loadout.InstallationId)
+            .Select(l => l.ShortName)
+            .ToArray()
+        );
+
+        var newName = "Loadout " + newShortName;
+
+        {
+            using var tx2 = Connection.BeginTransaction();
+            tx2.Add(newLoadoutId, Loadout.Name, newName);
+            tx2.Add(newLoadoutId, Loadout.ShortName, newShortName);
+
+            await tx2.Commit();
+        }
+
+        return Loadout.Load(Connection.Db, newLoadoutId);
```

Comment:
This would first create the copy with the wrong name, then update the name.

I would very much prefer to have a single commit.

What is stopping us from changing the names right before the commit for the first transaction?